src/app/shared/dialog-box/dialog-box.component.scss

```diff
@@ -0,0 +1,18 @@
+textarea {
+  width: 80% !important;
```

Comment:
!important is forbidden and we override all the textarea with this which is not what we want,
Let's remove that please.
We can make it a class or even better use the text-area and css classes provided by default if we look at the GDS govuk lib.